### ISSUE TITLE
Custom integration with UI?

### ISSUE THREAD

**Issue by reesericci** [Apipa169/Assistant-Relay-for-Hassio#41]
It would be cool if I could just add an integration from the UI once installed with HACS with a custom repo, or integrated. It would just ask me for the server and then it would create the services. Maybe add some discovery for the addon.

**Comment by Apipa169** [comment]
This will never work with HACS. Assistant Relay needs to run as add-on (docker container)

**Comment by reesericci** [comment]
No, the integration would just pair with the addon. That way we wouldn't need to add the rest_command to our configuration.yaml. It would be better because the service would be `assistant_relay.broadcast`, etc.

**Comment by reesericci** [comment]
@Apipa169 reopen?

**Comment by Apipa169** [comment]
Ah, sorry. Misunderstood your first post.

I have reopened the issue, but I'm quite busy at the moment so don't expect it soon. 

Or feel free to contribute :-)

**Comment by reesericci** [comment]
K
